src/NUnitFramework/framework/Internal/TestParameters.cs

```diff
@@ -135,12 +140,193 @@ public void ApplyToTest(Test test)
                 test.RunState = this.RunState;
 
             foreach (string key in Properties.Keys)
-                foreach (object value in Properties[key])
-                    test.Properties.Add(key, value);
+            foreach (object value in Properties[key])
+                test.Properties.Add(key, value);
         }
 
         #endregion
 
+        #region IAlignArguments Members
+        /// <inheritdoc />
```

Comment:
I'm pretty sure we don't do XML doc post-processing, so `<inheritdoc />` will do nothing in this codebase. .NET has no special understanding of this tag. @nunit/framework-team please correct me if I'm wrong.